attelo/decoding/control.py

```diff
@@ -60,10 +60,14 @@ def _predict_relate(dpack, models):
     and an list of labels
     """
     if models.relate == 'oracle':
-        idxes = dpack.target
+        unrelated = dpack.label_number(UNRELATED)
         # pylint: disable=no-member
-        probs = np.ones(idxes.shape)
+        # treat unrelated as unknown (to avoid cutting links that may
+        # have been inserted by an non-oracle attachment label)
+        always_related = np.vectorize(lambda x: 0 if x == unrelated else x)
```

Comment:
Hmm, OK. `s/x/lbl/` (maybe l if too long), `idxes/labels/` and indeed `s/probs/scores/`.
I kind of would prefer to hang on to `always_related` or in any case, a name that communicates a bit the low-level intent.  Any other suggestions for how I might rename this?